composer.json

```diff
@@ -39,7 +39,7 @@
     },
     "extra": {
         "branch-alias": {
-            "dev-master": "3.1-dev"
+            "dev-master": "3.3-dev"
```

Comment:
not sure about it. Do you really plan to maintain several branches ? For a bundle, it is probably fine to have a single maintained branch (except for multiple major version)